### PR TITLE
identify component if it is a registration

### PR DESCRIPTION
**Pull request by chennan47**
<b>Purpose</b>
Identify component registration. Closes [#OSF-4259].

<b>Changes</b>
before change:
screen shot 2015-09-03 at 3 45 07 pm

after change:
screen shot 2015-09-03 at 3 44 21 pm

### CODE REVIEW

**Comment by HarryRybacki** [comment]
@chennan47 CR1 looks good to me. Moving to CR2 :+1:

File: website/templates/search.mako [hunk 1]
```diff
@@ -235,8 +235,12 @@
       <div data-bind="template: {name: 'node', data: $data}"></div>
     </script>
     <script type="text/html" id="registration">
-        <h4><a data-bind="attr.href: url">{{ title }}</a>  (<!-- ko if: is_retracted --><span class="text-danger">Retracted</span> <!-- /ko -->Registration)</h4>
-
+        <!-- ko if: parent_url -->
+        <h4><a data-bind="attr.href: parent_url">{{ parent_title}}</a> / <a data-bind="attr.href: url">{{ title }}</a>  (<!-- ko if: is_retracted --><span class="text-danger">Retracted</span> <!-- /ko -->Registration)</h4>
+        <!-- /ko -->
+        <!-- ko if: !parent_url -->
+        <h4><span data-bind="if: parent_title">{{ parent_title }} /</span> <a data-bind="attr.href: url">{{ title }}</a>  (<!-- ko if: is_retracted --><span class="text-danger">Retracted</span> <!-- /ko -->Registration)</h4>
```
Comment (abought) [inline website/templates/search.mako]: This line of code uses two different form of knockout syntax for the `if` statement; I'm following up with @sloria to see if there are any style guidelines here, but this seems unnecessarily inconsistent.

**Comment by abought** [comment]
One question raised, but otherwise it runs and no issues noted.

Pass complete.

File: website/templates/search.mako (hunk 1 above)
Comment (sloria) [inline website/templates/search.mako]: I don't feel strongly either way.

Comment (sloria) [inline website/templates/search.mako]: But Andy makes a fair point about consistency on the same line. For both consistency and conciseness, why not use data-bind="if..." on the 2nd span?

Comment (chennan47) [inline website/templates/search.mako]: addressed in f5ba854

**Pull request merged**